RPG-game-Hung/main.py

```diff
@@ -0,0 +1,140 @@
+from termcolor import cprint
+from classes.game import Game
+from classes.db import Database
+from classes.colors import Color
+from classes.items import Item, ItemType
+from classes.magics import Magic, MagicType
+
+computer_names = [
+    "Hell Guard",
+    "Dark Lord",
+    "Evil Angel",
+    "Satan",
+    "Devil",
+]
+
+magics = {
+    1: Magic(name="Dark Love", dmg=25, mp_cost=15, type=MagicType.dark),
```

Comment:
If the `key` is something like an `index` number like this, why not using a `list` instead of a `dict`?